FSI/VFFS.md

```diff
@@ -3,21 +3,19 @@
 # Very Flexible Floating Structure (VFFS)
```

Comment:
This is already in the page title. If we remove this title, we can move the other sub-titles one level up, so they have a better highlighting.